agot-bg-game-server/src/messages/ServerMessage.ts

```diff
@@ -280,4 +282,9 @@ interface PlayerReplaced {
     type: "player-replaced";
     oldUser: string;
     newUser: string;
-}
\ No newline at end of file
+}
+
+interface CrowKillersStepChanged {
```

Comment:
@Longwelwind I only were able to properly update the UI with this new ServerMessage. All tries to force an update via listening to `game-state-change` have been unsuccessful. Ok for now? Should I do sth similar for PreemptiveRaid to fix the update issue there also and we continue tracking the technical improvement with #569?

Reply:
Yeah, ok for now. 🙂